src/main/java/com/epam/spring/homework3/service/repository/impl/PlaceRepositoryImpl.java

```diff
@@ -0,0 +1,46 @@
+package com.epam.spring.homework3.service.repository.impl;
+
+import com.epam.spring.homework3.service.model.Place;
+import com.epam.spring.homework3.service.repository.PlaceRepository;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Component
```

Comment:
Use @Repository annotation instead of @Component, it has actually no difference, but first one has more readability to reviewers